pkg/rule/builder.go

```diff
@@ -0,0 +1,251 @@
+package rule
+
+import (
+	"errors"
+	"net/netip"
+
+	"github.com/google/nftables"
+	"github.com/google/nftables/expr"
+	"github.com/ngrok/firewall_toolkit/pkg/expressions"
+	"golang.org/x/sys/unix"
+)
+
+type AddrFamily int8
+
+const (
+	AnyFamily AddrFamily = -1
+	IPv4      AddrFamily = unix.NFPROTO_IPV4
+	IPv6      AddrFamily = unix.NFPROTO_IPV6
+)
+
+type TransportProto int8
+
+const (
+	AnyTransport TransportProto = -1
+	TCP          TransportProto = unix.IPPROTO_TCP
+	UDP          TransportProto = unix.IPPROTO_UDP
+)
+
+type Verdict expr.VerdictKind
+
+const (
```

Comment:
Please add support for the all the possible verdicts. These don't change often and we might as well support all of them https://github.com/google/nftables/blob/main/expr/verdict.go#L42